### ISSUE TITLE
CSS class preventing custom field tab from being visible in post editor

### ISSUE THREAD

**Issue by geniuspro** [Obox/layerswp#261]
Hello. I want to add Custom Meta on my page options with video url.

```
add_filter(' layers_custom_meta ',' look_add_custom_meta ');
if( !function_exists( 'look_add_custom_meta' ) ) {
    function look_add_custom_meta($custom_meta) {
        $custom_meta['post'] = array (
            'title' => LAYERS_THEME_TITLE . __( ': Options' , 'layerswp' ),
            'description' => __( 'Meta Description' , 'layerswp' ), // @TODO
            'position' => 'normal',
            'custom-meta' => array(
                'media' => array(
                    'title' => __( 'Meta Title' , 'layerswp' ),
                     'elements' => array(
                         'video-url' => array(
                             'label' => __( 'Video URL' , 'layerswp' ),
                             'description' => __( 'For use with <a href="' . esc_url( 'http://codex.wordpress.org/' ) . 'Embeds" target="_blank">oEmbed</a> supported media' , 'layerswp' ),
                            'type' => 'text',
                        )
                    )
                )
            )
        );
    }
}
```

it not works. how can i do this?

**Comment by marcperel** [comment]
I think that the issue is that you're not returning the `$custom_meta` at the end @olziinaa, 

At the end of your function add: 

```return $custom_meta```

**Comment by olziinaa** [comment]
```
add_filter( 'layers_custom_meta', 'look_add_custom_meta' );
if( !function_exists( 'look_add_custom_meta' ) ) {
	function look_add_custom_meta( $custom_meta ){
		$custom_meta['post'] = array(
			'title' => LAYERS_THEME_TITLE . __( ': Options' , 'layerswp' ),
			'description' => __( '' , 'layerswp' ), // @TODO
			'position' => 'normal',
			'custom-meta' => array(
				'media' => array(
					'title' => __( 'Rich Media' , 'layerswp' ),
					'elements' => array(
						'video-url' => array(
							'label' => __( 'Video URL' , 'layerswp' ),
							'description' => __( 'For use with <a href="' . esc_url( 'http://codex.wordpress.org/' ) . 'Embeds" target="_blank">oEmbed</a> supported media' , 'layerswp' ),
							'type' => 'text',
						)
					)
				),
				'title' => array(
					'title' => __( 'Title Options' , 'layerswp' ),
					'elements' => array(
						'pd-top' => array(
							'label' => __( 'Padding Top' , 'layerswp' ),
							'description' => __( 'Padding Top Size' , 'layerswp' ),
							'type' => 'text',
						),
						'pd-bottom' => array(
							'label' => __( 'Padding Bottom' , 'layerswp' ),
							'description' => __( 'Padding Top Size' , 'layerswp' ),
							'type' => 'text',
						)
					)
				)
			)
		);
		// Page Meta we just emulate the post meta
		$custom_meta['page'] = $custom_meta['post'];
		return $custom_meta;
	}
}
```

Thanks again for the help me. I want to add some options for page on my child theme. Created second tab on custom meta field but it not shows me.

**Comment by olziinaa** [comment]
Maybe it is little bug on layers theme.

**Comment by olziinaa** [comment]
Hello @marcperel 
I found solution for this bug. 
Go to the layerswp\core\assets\admin.js line 396

add this code 
```
$tab_container.find( 'section.layers-tab-content' ).eq( $i ).addClass('layers-show').removeClass('layers-hide').slideDown().siblings( 'section.layers-tab-content' ).addClass('layers-hide').removeClass('layers-show').slideUp();
```
after this problem works fine.

Thanks for the amazing theme layers.

**Comment by marcperel** [comment]
Thanks @olziinaa we'll check that out in the next release, is the issue that the tabs would not work when clicked?

**Comment by olziinaa** [comment]
Yes, Tab content is not show when click the tab.

**Comment by marcperel** [comment]
ta!

**Comment by geniuspro** [comment]
Hello again. how can i update custom meta default value.

**Comment by geniuspro** [comment]
When will resolve this bug?

**Comment by vailjoy** [comment]
Hi there,
Please see the following for how to create custom fields in your own meta box :+1: http://docs.layerswp.com/how-to-add-custom-fields-to-posts-and-pages/

 The above is slated for 1.2.5, we do not have an ETA

**Comment by vailjoy** [comment]
Should be fixed in 1.2.14
